.github/ISSUE_TEMPLATE/Bug-Report.yml

```
@@ -0,0 +1,46 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please check the [devstack troubleshooting guide](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/troubleshoot_general_tips.html) and the [existing list of blocking bugs](https://github.com/openedx/devstack/labels/blocker) before filing a new issue.
+  - type: textarea
+    id: bug-report
+    attributes:
+      label: Describe the bug that you are seeing.
+    validations:
+      required: true
+  - type: input
+    id: container
+    attributes:
+      label: Did this happen on your machine or in the container?
+      description: e.g. Did this happen outside of running `make dev.shell.<service>` or inside running `make dev.shell.<service>`?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce.
+      description: How could someone replicate what you're seeing?
+    validations:
+      required: true
+  - type: dropdown
+    id: mac-type
+    attributes:
+      label: Apple Silicon Mac?
+      description: Are you running into this issue on an Apple Silicon Mac?
```

1. Can we ask questions that work whether someone is self-reporting an issue, or if we (or someone) is adding an issue after the fact that more than one person may have seen? For example:
```suggestion
      description: Was this issue seen on an Apple Silicon Mac?
```
2. Should we **also** ask whether the issue was seen on an Apple Intel, or any non-Apple Silicon machine? Note that we may end up adding this data later, or we may have multiple cases by the time the ticket is created.

---

Yeah, I think we can get away with a more nuanced answer with dropdowns. I'll think about how to structure this some more.